content/doc/developer/security/index.adoc

```diff
@@ -96,6 +98,63 @@ To do this, write Jelly like this:
 NOTE: This is not to be confused with the +checkPermission+ invocation in your operation.
 Users can still hit the URL directly, so you still need to protect the operation itself, in addition to disabling the UI rendering
 
+=== Authentication methods
+
+In Jenkins the security engine that is used is Acegi Security (ancestor of Spring Security).
+Without any special plugins to manage authentication, an instance of Jenkins is packaged
+with the following authentication methods:
+
+* Web UI
+** When you use the form on the login page, using the fields +j_username+ and +j_password+
+* REST API
+** Using Basic with login / *password*
+** Using Basic with login / *apiToken*
+* Jenkins-CLI jar
+** (deprecated) using remoting transport with login / logout command
+** (deprecated) username / password as parameters on each command
+** +-auth+ argument with username:*password* or username:*apiToken* that will do something like HTTP calls
+** using SSH transport mode with your local keys
+* CLI over SSH
+** directly using the native SSH command, without Jenkins-CLI
```

Comment:
If you want to mention all authentication methods, `QueueItemAuthenticator` should be in the lost IMHO